docs/community/3.15-announcement.md

```diff
@@ -0,0 +1,58 @@
+<style>
+.promo li a {
+    float: left;
+    width: 130px;
+    height: 20px;
+    text-align: center;
+    margin: 10px 30px;
+    padding: 150px 0 0 0;
+    background-position: 0 50%;
+    background-size: 130px auto;
+    background-repeat: no-repeat;
+    font-size: 120%;
+    color: black;
+}
+.promo li {
+    list-style: none;
+}
+</style>
+
+# Django REST framework 3.15
+
+At the Internet, on January 12th, 2024, with 135 commits 107 authors, we are happy to announce the release of Django REST framework 3.15.
+
+## Django 4.2 and Python 3.11 support
+
+The latest release now fully supports Django 4.2 and Python 3.11.
```

Comment:
The latest release now fully supports Django 4.2 and Python 3.12.

Reply:
python 3.12 still has failures

Reply:
OK, my intention was leave it ready for when it's done

Reply:
I sense that Py3.12 will work but not Dj5.

Reply:
now it supports py 3.12 and dj 5.0